libpdbg/target.c

```diff
@@ -155,6 +155,32 @@ int pib_read(struct pdbg_target *pib_dt, uint64_t addr, uint64_t *data)
 	return rc;
 }
 
+int pib_ody_read(struct pdbg_target *pib_dt, uint64_t addr, uint64_t *data)
+{
+	struct pib *pib;
```

Comment:
<img width="1135" alt="image" src="https://github.com/open-power/pdbg/assets/112170550/bafbc096-8136-46d0-849e-65dbc6376775">

Reply:
it is like upcasting to original pib pointer so it can never be NULL, following same as pib_read method